CLI/controllers/sendSchemaController.go

```diff
@@ -20,106 +20,86 @@ func serialiseVector(attributes map[string]interface{}, attributeName string) {
 	}
 }
 
-// Serialising size & posXY is inefficient but
-// the team wants it for now
-// "size":"[25,29.4,0]" -> "size": "{\"x\":25,\"y\":29.4,\"z\":0}"
 func serialiseStringVector(attr map[string]interface{}, want string) string {
 	var newSize string
-	if size, ok := attr[want].(string); ok {
-		left := strings.Index(size, "[")
-		right := strings.Index(size, "]")
-		coords := []string{"x", "y", "z"}
 
-		if left != -1 && right != -1 {
-			var length int
-			subStr := size[left+1 : right]
-			nums := stringSplitter(subStr, ",", want)
-			if nums == nil { //Error!
-				return ""
-			}
-			//nums := strings.Split(subStr, ",")
+	size, ok := attr[want].(string)
```

Comment:
is it always size, though?

Reply:
Not always... I've changed it to "vector".